packages/experimental/Checkbox/src/useCheckbox.ts

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import {
+  useAsPressable,
+  useKeyCallback,
+  useOnPressWithFocus,
+  useViewCommandFocus,
+  useAsToggle,
+} from '@fluentui-react-native/interactive-hooks';
+import { CheckboxProps, CheckboxInfo, CheckboxState } from './Checkbox.types';
+import { IPressableProps } from '@fluentui-react-native/pressable';
+
+export const useCheckbox = (props: CheckboxProps): CheckboxInfo => {
```

Comment:
Maybe add some documentation on this method so when people hover over it, VS code / intelligence gives a short summary? 
Note: I only recently learned about how we can use hooks to provide the state of an _entire_ component with one hook, so this pattern was confusing to me at first. Now that I understand it, I think it's pretty cool :D